device/esp_tinyusb/test_apps/cdc_and_usb_device/main/test_cdc.c

```diff
@@ -42,10 +42,29 @@ static const tusb_desc_device_t cdc_device_descriptor = {
 static const uint16_t cdc_desc_config_len = TUD_CONFIG_DESC_LEN + CFG_TUD_CDC * TUD_CDC_DESC_LEN;
 static const uint8_t cdc_desc_configuration[] = {
     TUD_CONFIG_DESCRIPTOR(1, 4, 0, cdc_desc_config_len, TUSB_DESC_CONFIG_ATT_REMOTE_WAKEUP, 100),
+#if (TUD_OPT_HIGH_SPEED)
+    TUD_CDC_DESCRIPTOR(0, 4, 0x81, 8, 0x02, 0x82, 512),
```

Comment:
There is a way to simplify these lines with ternary operator: 
```suggestion
    TUD_CDC_DESCRIPTOR(0, 4, 0x81, 8, 0x02, 0x82, (TUD_OPT_HIGH_SPEED ? 512 : 64),
```

Reply:
Ye, I was thinking about doing it exactly this way. But did not do it in the end.. Will fix it now. Thx for checking.